test/lan_test.rb

```diff
@@ -159,3 +159,37 @@
     expect(Yast::LanItems.Items).to be_empty
   end
 end
+
+describe "LanClass#Modified" do
+  def expect_modification_succeedes(modname, method)
+    allow(modname)
+      .to receive(method)
+      .and_return true
+
+    expect(modname.send(method)).to be true
```

Comment:
Here you mock a method and then you call it. It does succeed but it is not testing any of production code.